backend/app/services/jobs.py

```
@@ -156,23 +198,280 @@ def repo_update_worker(stop_event: threading.Event):
+                    try:
+                        count = loop.run_until_complete(
+                            summary_service.send_daily_summary(
+                                db, hours_back=hours_back
+                            )
+                        )
+                        logger.info(
+                            f"[job] Daily group chat summary completed, sent {count} emails"
+                        )
+                    finally:
+                        db.close()
+                except Exception as e:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)